### PR TITLE
Revert "Revert "Updated xcconfigs to new official repository""

### PR DESCRIPTION
**Pull request by alvinchongry**
revert-12-revert-11-develop

### CODE REVIEW

**Review by alvinchongry — COMMENTED** [review]
7c1a6c50bb86daaab51c9e141bd8066bd5aff565

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "Carthage/Checkouts/xcconfigs"]
 	path = Carthage/Checkouts/xcconfigs
-	url = https://github.com/jspahrsummers/xcconfigs.git
+	url = https://github.com/xcconfigs/xcconfigs.git
```
Comment (alvinchongry) [inline .gitmodules:3]: GPG key ID: 4AEE18F83AFDEB23

**Review by alvinchongry — COMMENTED** [review]
GPG key ID: 4AEE18F83AFDEB23

**Comment by ekscrypto** [comment]
What are you doing?  Are you just generating GitHub traffic?

**Review by alvinchongry — COMMENTED** [review]
7c1a6c50bb86daaab51c9e141bd8066bd5aff565

**Pull request merged**